exercises/grade-school/grade_school_test.rb

```diff
@@ -4,85 +4,104 @@
 require_relative 'grade_school'
 
 class SchoolTest < Minitest::Test
-  attr_reader :school
-
-  def setup
-    @school = School.new
-  end
-
-  def test_an_empty_school
-    assert_equal({}, school.to_h)
+  def test_empty_grade
+    subject = School.new
```

Comment:
I think a "list" in this context should be a list of students, such as

```text
"Adam\n
Betty\n
Charlie\n
Denise\n"
```
rather than

```ruby
[
  'Adam",
  "Betty",
  "Charlie",
  "Denise",
]
```

Then the implementation is left open, we only test for the result of the query, rather than the structure.